doc/api/net.md

```diff
@@ -343,23 +351,21 @@ added: v0.3.4
 
 Construct a new socket object.
 
-`options` is an object with the following defaults:
-
-```js
-{
-  fd: null,
-  allowHalfOpen: false,
-  readable: false,
-  writable: false
-}
-```
-
-`fd` allows you to specify the existing file descriptor of socket.
-Set `readable` and/or `writable` to `true` to allow reads and/or writes on this
-socket (NOTE: Works only when `fd` is passed).
-About `allowHalfOpen`, refer to [`net.createServer()`][] and [`'end'`][] event.
-
-`net.Socket` instances are [`EventEmitter`][] with the following events:
+* `options` {Object} Available options are:
+  * `fd`: {number | null} Defaults to `null`. If specified, wrap around an
+    existing socket with the given file descriptor.
+  * `allowHalfOpen` {boolean} Default to `false`. Indicates whether
+    half-opened TCP connections are allowed. See [`net.createServer()`][]
+    and [`'end'`][] event for details.
+  * `readable` {boolean} Default to `false`. Works only when `fd` is passed.
```

Comment:
suggest: "Allow reads on the socket when a `fd` is passed, otherwise ignored." (same below)